### ISSUE TITLE
Issue on README.md

### ISSUE THREAD

**Issue by orklann** [google/dart-gl#19]
on README.md, it worte: `pub run tools/gl_generate.dart`, but the dart file name in `tools` directory is `gl_generator.dart`, typo, right?

**Comment by jtmcdole** [comment]
Yep, that's a typo.  I actually wanted to just name it "generator.dart" so that the instructions are nearly the same between dart:gl and dart:glfw.
